tools/serverpod_cli/lib/src/generator/dart/library_generators/util/model_filter_util.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:path/path.dart' as p;
+
+import 'package:serverpod_cli/src/analyzer/models/definitions.dart';
+import 'package:serverpod_cli/src/config/config.dart';
+
+List<ClassDefinition> getSealedClasses(
```

Comment:
Suggestion: Rename to `getSealedTopNodeClasses`.

Can this method also be private?